### DISCUSSION TITLE
Display support in precompiled binaries

### DISCUSSION THREAD

**Discussion by Poimander** [arendst/Tasmota#12763] [Q&A] [answered]
Hi,
is it possible to add display support (OLED Display Module 0.66" - Wemos shield) in precompiled binaries (tasmota.bin, tasmota-sensors) instead of using "tasmota-displays-bin" (without power meter support)?

Thank you very much!

**Comment by barbudor** [comment, marked as answer]
The whole purpose of Tasmota predefined binaries is to have specialized, fine tuned, focused binaries.

If you want to combine different features set, you have to compile your own binary and may be have to tune it. You have multiple tools to do so from online TasmoCompiler, Gitpod or local VisualStudioCode with PlatformIO.

Please read the doc and strictly follow the recommendation that a custom firmware should be derived from "tasmota" and not from any variant.
Say that you want to have a given sensor and a given display in the same binary, you must add that sensor and that display to "tasmota" and not add display to "tasmota-sensors" or vice-versa.

You can also check extended binaries built by te team but considered as unsupported here: https://github.com/tasmota/binaries

**Reply by Poimander** [reply]
I really thank you: you clarified an aspect in which I was stuck! Thank you very much!
